boards/arm/nucleo_f746zg/doc/nucleof746zg.rst

```diff
@@ -0,0 +1,189 @@
+.. _nucleo_f746zg_board:
+
+ST Nucleo F746ZG
+################
+
+Overview
+********
+
+The STM32 Nucleo-144 boards provide an affordable and flexible way for users to try out new concepts and build prototypes by choosing from the various combinations of performance and power consumption features, provided by the STM32 microcontroller. For the compatible boards, the SMPS significantly reduces power consumption in Run mode.
+
+The ST Zio connector, which extends the Arduino™ Uno V3 connectivity, and the ST morpho headers provide an easy means of expanding the functionality of the Nucleo open development platform with a wide choice of specialized shields.
+
+The STM32 Nucleo-144 board does not require any separate probe as it integrates the ST-LINK/V2-1 debugger/programmer.
+
+The STM32 Nucleo-144 board comes with the STM32 comprehensive free software libraries and examples available with the STM32Cube MCU Package.
+
+Key Features
+
+    STM32 microcontroller in LQFP144 package
+    External SMPS to generate Vcore logic supply (only available on '-P' suffixed boards)
+    Ethernet compliant with IEEE-802.3-2002 (depending on STM32 support)
+    USB OTG or full-speed device (depending on STM32 support)
+    3 user LEDs
+    2 user and reset push-buttons
+    32.768 kHz crystal oscillator
+
+    Board connectors:
+
+    -   USB with Micro-AB
+    -   SWD
+    -   Ethernet RJ45 (depending on STM32 support)
+    -   ST Zio connector including Arduino™ Uno V3
+    -   ST morpho
+
+    Flexible power-supply options: ST-LINK USB VBUS or external sources
+    On-board ST-LINK/V2-1 debugger/programmer with USB re-enumeration capability: mass storage, virtual COM port and debug port
+    Comprehensive free software libraries and examples available with the STM32Cube package
+    Support of a wide choice of Integrated Development Environments (IDEs) including IAR™, Keil®, GCC-based IDEs, Arm® Mbed™
```

Comment:
We recognize trademarks of other companies by using an asterisk ``*`` and not a TM or (r) symbol.